Google.Api.Generator.Tests/WhitespaceFormatterTest.cs

```diff
@@ -237,13 +237,43 @@ public void ForEach(IEnumerable<string> strings)
             private void AsyncLambdaParam(Func<Task> fn) => throw new NotImplementedException();
 
             public void AsyncLambda() => AsyncLambdaParam(async () => await Task.Yield());
+
+            // Test switch statement
+            public int Switch(int i)
+            {
+                switch (i)
+                {
+                    // Single-line case.
+                    case 0: return 1;
+                    case 1: return 0;
+                    // Multi-line case.
+                    case 1111:
+                        int j = Switch(0);
+                        return i + j;
+                    default: return i;
+                }
+            }
         }
 
         // Test base-list with one item.
         private class EmptyInner : List<string>
         {
             // Test class that has no members.
         }
+
+        // Test enum.
+        public enum AnEnum
+        {
+            // Enum members with value.
+            A = 1,
+
```

Comment:
I'm not sure the blank lines here are adding to the readability.
Also, would it be worth including tests with XML comments? That's the most likely scenario.

Reply:
In general in generated code, I think it is better to have blank lines between enum entries, especially given that there will almost always be XmlDoc present.
Unfortunately this whitespace testing doesn't currently support XmlDoc (or any form of comments), so it can't test with xmldoc currently.
I could add this, but would rather do in a future PR.